app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
@tenderlove @chessbyte Current code works if I switch over to .destroy!, then the begin/rescue is able to trap assigned rate NON delete and issue proper messages in UI. Just tried it with current spec test and it seems to work. Do we have a need for the run_callbacks? I can put that in and change code to start there.

Not sure which way to go with the code at this point.